platform/error/platform-error-impl/src/main/java/org/codice/ddf/platform/error/handler/impl/DefaultErrorHandler.java

```diff
@@ -13,17 +13,16 @@
  */
 package org.codice.ddf.platform.error.handler.impl;
 
-import static org.boon.Boon.toJson;
```

Comment:
❓ If we removed boon does that mean we can remove the dependency from the pom or do other things in here still use it?

Reply:
Good catch. I should be able to remove that dependency.

Reply:
Removed boon dependency.